PROJETOS/20251105-github-actions-workflow-modifier/real-autorelease.yaml

```
@@ -0,0 +1,129 @@
+    tags:
+    - '*'
+  pull_request:
+    branches:
+    - main
+  workflow_dispatch:
+    inputs:
+      new_version:
+        description: 'New tag version'
+        default: 'patch'
+  schedule:
+    - cron: '0 2 * * 6' # saturday 2am
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)